src/sentry/static/sentry/app/views/performance/transactionSummary/transactionList.tsx

```diff
@@ -146,6 +148,83 @@ class TransactionTable extends React.PureComponent<Props> {
     });
   };
 
+  handleCellAction = (column: TableColumn<React.ReactText>) => {
+    return (action: Actions, value: React.ReactText) => {
+      const {eventView, location} = this.props;
+
+      let nextLocationQuery: Location['query'] = {};
+      const searchConditions = tokenizeSearch(eventView.query);
```

Comment:
This will have a logical conflict with Evan's pull request that adds OR conditions. I don't think he'll get a git conflict, but the merge result of his and this branch will be broken. If you merge first could you update his branch with the new implementation of this?

Reply:
I think the smoothest path forward will be to rebase this PR on top of Evans and make the necessary changes there.